jobs/gtfs-rt-parser-v2/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Andrew Vaccaro <atvaccaro@gmail.com>"]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.10"
-calitp = "2022.8.19a0"
+calitp = "2022.9.1a3"
```

Comment:
this is ok for now, I think that ideally middle of next week we get everyone (RT, schedule, validation, raw) on an actual full release. so I would also be ok with `8.25` since that's what you did for schedule validation, or we push a new full release of calitp next week and use that everywhere

Reply:
Ok cool post https://github.com/cal-itp/data-infra/pull/1740 I will update the infra repo broadly